File: Work2/8.py
# Двойной факториал числа
print('Input n:')
n = int(input())
def doublefact (n):
    if (n==0 or n==1):
        return 1
    else:
        return n*doublefact(n-2)
    
print ("Double factorial of", n, "=", doublefact(n))

File: Work2/8.cpp
#include <iostream>
using namespace std;
int fact (int n) {
if (n==0 || n==1) {
    return 1;
}
else {
    return n*fact(n-2);
}
}
int main (){
int n;
cout << "Input n"<<endl;
cin >> n;
int dfactorial = fact(n);
cout << "Double factorial = " << dfactorial << endl;
}
